matminer/featurizers/site.py

```diff
@@ -33,6 +36,18 @@
 
 from matminer.featurizers.stats import PropertyStats
 
+cn_motif_op_paras = {}
+with open(os.path.join(os.path.dirname(
+        pymatgen.analysis.__file__), 'cn_opt_paras.yaml'), 'r') as f:
+    cn_motif_op_paras = yaml.safe_load(f)
+    f.close()
```

Comment:
if you use ``with open()``, you don't need to do ``f.close()`` (that's the primary point of using with open() instead f.open())